extensions/wikia/Announcements/models/Announcements.class.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+class Announcements {
+
+	/**
+	 * @desc Get users who edited or contributed on specific wiki in specific amount of time
+	 *
+	 * @param $wikiId
+	 * @param $period
+	 *
+	 * @return mixed
+	 */
+	public function getActiveUsers( int $wikiId, int $period ) {
+		global $wgSpecialsDB;
+
+		$db = wfGetDB( DB_SLAVE, [], $wgSpecialsDB );
+		$userIds = [];
+
+		$query = $db->select(
```

Comment:
`selectFieldValues` helper allows you to immediately fetch values of user_id column into an array